test/e2e/tests/test_network/test_network.py

```diff
@@ -61,7 +61,7 @@ def expected_events(origin: dict, destination: dict) -> list:
             "evt.type": "accept4",
             "fd.name": None,
             "proc.cmdline": "nginx",
-            "proc.exe": "nginx: master proces",
+            "proc.exe": "nginx: master process nginx -g daemon off;",
```

Comment:
@Andreagit97 how is this possible that we haven't caught this before?

Reply:
I realised this too late: that was an expected failure on the modern probe. I just removed the commit. I think we are good to go!

Reply:
This is our man https://github.com/falcosecurity/libs/pull/976